scripts/remove-completed-cases-from-action/1_get_case_ids_of_completed_actionable_cases_to_csv.sql

```diff
@@ -0,0 +1 @@
+\copy (SELECT id FROM casesvc.case WHERE statefk = 'ACTIONABLE' and casegroupfk in (SELECT casegrouppk FROM casesvc.casegroup WHERE status = 'COMPLETEDBYPHONE' or status = 'NOLONGERREQUIRED' or status = 'COMPLETE')) TO STDOUT WITH CSV
```

Comment:
Do we care about these cases being in an "ACTIONABLE" state? I'd have though we can just remove all cases that are in any of the completed case group statuses